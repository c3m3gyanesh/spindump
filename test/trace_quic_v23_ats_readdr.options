--report-notes --report-spin-flips --format text
