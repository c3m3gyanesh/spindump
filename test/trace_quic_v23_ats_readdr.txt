A test of the ATS QUIC implementation, using address migration. The client is Lsquic, the server is ATS. The following Lsquic command was used:

    . /lsquic/http_client -H quic.ogre.com -s quic.ogre.com:4443 -p /en/latest/admin-guide/introduction.en.html
