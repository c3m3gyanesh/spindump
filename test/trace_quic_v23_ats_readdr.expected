QUIC 172.16.114.251 <-> 67.192.246.159 088d77e7d72627ce-206d105c7b6c3afdc07f349bcfbe6c557f2287 (54753:4443) at 1571173106837774 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 088d77e7d72627ce-12d72c618025c7685b253306c28907b50267 (54753:4443) at 1571173107051319 measurement up right 61400 packets 2 0 bytes 2456 0 bandwidth 2456 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 8d77e7d72627ce69-12d72c618025c7685b253306c28907b50267 (54753:4443) at 1571173107407780 spinflip up 0-1 initiator packets 11 8 bytes 3533 8152 bandwidth 3533 8152 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 67.192.246.159 null-null (54753:4433) at 1571173107610029 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xfffffff"
